Update Customer SDK docs in preparation for stable release (#693)
livechat/livechat-public-docs: content/extending-ui/extending-chat-widget/v2.0/customer-sdk/index.mdx

# Changelog

## [v2.0.0-beta.3] - xxxx-xx-xx

### Changed

- The iterator returned from `getChatHistory` no longer returns `users`. Their data needs to be obtained independently, for example by using `getChat` method.
- `listChats` no longer accepts `offset` parameter. Instead, it now accepts `pageId` to which you can provide a value of `previousPageId` or `nextPageId` returned from other `listChats` calls.
- Chat objects returned from `listChats` no longer have `lastThreadOrder` property, but they have received a new `lastThreadCreatedAt` property instead.
- `threads` objects no longer have `order` property, but they have received a new `createdAt` property instead.
- `eventsSeenUpToMap` is no longer available on thread object, it has been moved to chat objects.

> - The payload of the `chat_transferred` event has been changed. It now always have a `reason` property, contains `queue` property if the chat has been transferred and queued immediately and `requesterId` is available when `reason` is equal to `"manual"`.

Reviewer: ```suggestion
- The payload of the `chat_transferred` event was changed. It now always has the `reason` property. It contains the `queue` property if the chat has been transferred and queued immediately. `requesterId` is available when `reason` is equal to `"manual"`.
```
